initial-commit-tf.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+TF_REPO_NAME=dataworks-repo-template-terraform
+NEW_REPO_NAME=$1
+
+git config --global user.name "${GIT_USERNAME}"
+git config --global user.email "${GIT_EMAIL}"
+
+git clone https://github.com/dwp/$NEW_REPO_NAME
+cd $NEW_REPO_NAME
+
+if [ ! -z 'grep -rnw $TF_REPO_NAME ci' ]; then
+    git checkout -b initial-commit
```

Comment:
If CI is doing this then don't create a branch, it should commit to master

Reply:
Ah nice 👍